charts/ChartConfig.tsx

```diff
@@ -61,10 +61,11 @@ import {
     Time,
     TimeBounds
 } from "./TimeBounds"
+import detectNode from "detect-node"
 
 declare const App: any
 declare const window: any
-const isNode: boolean = require("detect-node")
+const isNode: boolean = detectNode
```

Comment:
Nice. Should we just have our own isNode check? Something like `typeof exports !== "undefined"` instead of using a package?

Reply:
https://github.com/iliakan/detect-node/blob/563e0b838ec1dd9b169d843268cdb220b78ddd91/index.js#L1-L2
That node module is very simple indeed 😄